FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -204,6 +205,10 @@ - (NSString *)FIRAppNameFromFullyQualifiedNamespace {
 /// requests to work.(b/14751422).
 - (void)refreshInstallationsTokenWithCompletionHandler:
     (FIRRemoteConfigFetchCompletion)completionHandler {
+  if (self.testWithoutNetwork) {
```

Comment:
I wonder what makes this flag necessary? Would a test utils with mock dependencies like in [FCM](https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Tests/UnitTests/FIRMessagingTestUtilities.h) solve the issue without changes to the production code?

Reply:
Eliminated the `testWithoutNetwork ` flag by mocking `refreshInstallationsTokenWithCompletionHandler:`. Mocking the dependencies seems more complicated and may be better to do in conjunction with a bigger refactor of RCNFetch.